interface-definitions/include/interface-mirror.xml.i

```diff
@@ -1,11 +1,31 @@
 <!-- included start from interface-mirror.xml.i -->
-<leafNode name="mirror">
+<tagNode name="mirror">
   <properties>
     <help>Incoming/outgoing packet mirroring destination</help>
     <completionHelp>
       <script>${vyos_completion_dir}/list_interfaces.py</script>
     </completionHelp>
-    <multi/>
   </properties>
-</leafNode>
+  <children>
+    <leafNode name="direction">
```

Comment:
How to monitor both `ingres` and `egres`?

Reply:
Use the following command:
```
set interfaces ethernet eth0 mirror lo
```
By default, mirror two-way traffic